wls-gui-wahllokalsystem/src/components/wahlvorbereitung/WahlumgebungUwbCard.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title>Zahl der Wahlurnen</v-card-title>
+      <v-card-text class="pb-0">
+        <v-form
+          ref="wahlurnenForm"
+          v-model="anzahlWahlurnenValidForm"
+        >
+          <div class="d-flex flex-wrap justify-start">
+            <div
+              v-for="(wahl, index) in urnenwahlVorbereitung.urnenAnzahl"
+              :key="index"
+            >
+              <v-number-input
+                v-model="wahl.anzahl"
+                class="mr-4"
+                :rules="[REQUIRED, MIN_NUMBER(1), MAX_NUMBER(99)]"
+                :data-test="`textFieldUrnenAnzahl_${index}`"
+                :label="`Anzahl der Wahlurnen ${getWahlNameById(wahl.wahlID)}`"
+                min-width="20rem"
+                clearable
+              />
+            </div>
+          </div>
+          <v-checkbox
+            v-model="isCheckboxAlleVersiegeltEnabled"
+            :label="checkboxLableText"
+            data-test="checkboxAlleVersiegelt"
+          />
+        </v-form>
+      </v-card-text>
+    </v-card>
+
+    <v-card class="mt-2">
+      <v-card-title>Abstimmungsschutzvorrichtungen</v-card-title>
+      <v-card-text class="pb-0">
```

Comment:
```suggestion
      <v-card-text class="pb-0 pt-2">
```

hier könnte man sich auch überlegen, die v-cards zentral zu stylen